src/installer.py

```diff
@@ -6,17 +6,42 @@
 sys.path.append('deps')
 sys.path.append('../deps')
 import pexpect
+import subprocess
+import urllib.request
 
 import password_type
 
 class Installer:
+    click_rootdir = os.getcwd()
+
+    def needs_custom_images(self) -> bool:
+        if os.uname().machine != "aarch64": # images only compiled for arm64
```

Comment:
This might break for aarch64 devices with an armhf userspace iirc? But I don't have one to verify

Reply:
Considering the images themselves contain `arm64` binaries still I don't believe they would work anyway, official support including for ARM32 userspace would bring support later I guess. This is mostly for the newly supported ARM64 Halium 12+ devices like Volla Phone X23 and others down the line

Reply:
Ah I think you're on about https://github.com/waydroid/waydroid/commit/807341e, I could also add it to be sure if you'd like